src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -59,7 +59,7 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
 
-                Ssl.SetProtocolOptions(innerContext, protocols);
+                Ssl.SetProtocolOptions(innerContext.DangerousGetHandle(), protocols);
```

Comment:
Please add a commend describing why this is safe.  I.e., what's keeping the handle open for the duration of this call?